src/lib/utils/hooks/useLongPress.ts

```diff
@@ -0,0 +1,70 @@
+import { useCallback, useRef, useState } from "react";
```

Comment:
Can you please provide some clear comments explaining what is happening in this hook?

Reply:
Added few comments regarding main logic of the hook.